modules/access_level/variables.tf

```diff
@@ -16,68 +16,75 @@
 
 variable "name" {
   description = "Description of the AccessLevel and its use. Does not affect behavior."
+  type        = string
 }
 
 variable "policy" {
   description = "Name of the parent policy"
+  type        = string
 }
 
 variable "combining_function" {
   description = "How the conditions list should be combined to determine if a request is granted this AccessLevel. If AND is used, each Condition must be satisfied for the AccessLevel to be applied. If OR is used, at least one Condition must be satisfied for the AccessLevel to be applied."
+  type        = string
   default     = "AND"
 }
 
 variable "description" {
   description = "Description of the access level"
-  default     = ""
+  type        = string
```

Comment:
Why are we removing the default? No reason to start requiring this.

Reply:
@morgante , you're absolutely right. 
Good eye! Thank you.

Reply:
@morgante, fixed [here](https://github.com/terraform-google-modules/terraform-google-vpc-service-controls/compare/c5b4156eb1518cee8d14e853ff2a3c2817e7fbd3..6f5f3941c9eb341d09fdcfb9e36bf204341f31ed#diff-bf717ad8c19cb9d311dbbee75674f072R36)
Thank you for review!